src/learning-catalogue/model/user.ts

```diff
@@ -0,0 +1,79 @@
+import {LineManager} from './lineManager'
+
+export class User {
+	static create(data: any) {
```

Comment:
User isn't a learning catalogue object, it's a CSRS object? Weren't we going to call this Profile?

Reply:
Removed. This wasn't being used and was changed to Identity in a previous poeceof work.

Reply:
Removed. This wasn't being used and was changed to Identity in a previous piece of work.